e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/your-buyer/summary-list/check-your-answers-your-buyer-summary-list.spec.js

```diff
@@ -115,4 +117,12 @@ context('Insurance - Check your answers - Your buyer page - Summary list', () =>
   it(`should render a ${FAILED_PAYMENTS} summary list row`, () => {
     checkSummaryList[FAILED_PAYMENTS]();
   });
+
+  it(`should not render a ${HAS_PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER} summary list row`, () => {
+    checkSummaryList[HAS_PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER]({ shouldRender: false });
+  });
+
+  it(`should not render a ${PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER} summary list row`, () => {
+    checkSummaryList[PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER]({ shouldRender: false });
+  });
```

Comment:
Should we update the E2E test description, it's not clear to me why these rows should not render in this scenario